### PR TITLE
disable early access feature snapshot storage

### PR DESCRIPTION
**Pull request by realsnick**
via issue https://github.com/hyperledger/besu/issues/4764

### CODE REVIEW

File: besu.yml [hunk 1]
```diff
@@ -74,9 +74,9 @@ services:
       - --metrics-port
       - "6060"
       - --data-storage-format=BONSAI
-      - --Xbonsai-use-snapshots=true
-#      - --sync-mode=X_CHECKPOINT
-      - --sync-mode=X_SNAP
+      - --Xbonsai-use-snapshots=false
+      - --sync-mode=X_CHECKPOINT
```
Comment (macfarla) [inline besu.yml:78]: did you mean to also change the sync-mode?

Comment (realsnick) [reply besu.yml:78]: yes, snapshotting is still alpha

Comment (realsnick) [reply besu.yml:78]: my node is almost sync, i'll report if this fixed the issue. I believe it did but let me 100% make sure

Comment (macfarla) [reply besu.yml:78]: 👍

Comment (realsnick) [reply besu.yml:78]: I can confirm this fixed the issue

**Review by realsnick — COMMENTED** [review]
works

**Comment by yorickdowne** [comment]
Checkpoint sync is plain worse than snap sync on stability.

As for bonsai snapshots, damned if you do damned if you don't: They were enabled because of https://github.com/hyperledger/besu/issues/4553

I'd like you to get the gitpoap for this one. If you'll just comment out bonsai snapshots and leave the rest as-is, I'll approve.

**Comment by realsnick** [comment]
Question is if snap sync is alpha or not as for me it was not able to recover and dropped my validator below 50% uptime until I resolved it with checkpoint . 

Maybe will be good idea to add this as an option for default.env?

**Comment by yorickdowne** [comment]
Bonsai snapshots certainly have issues. Snap sync itself, the only issue I am aware of is that sometimes it doesn't finish initial sync.

If checkpoint works better for you, I'd say make that change locally and `git commit` it so it stays.

Ultimately, Besu needs to stabilize its sync.

**Pull request merged**